### DISCUSSION TITLE
Version PDF %PDF-3.0\n3 0

### DISCUSSION THREAD

**Discussion by DiegoFSFlor** [empira/PDFsharp#195] [Q&A] [answered]
In the pdf that I'm trying to load, it has the version: %PDF-3.0\n3 0, in the GetPdfFileVersion calculation it is the value num2 with the value 30, so it returns 0, how do you fix it?

**Comment by TH-Soft** [comment, marked as answer]
We don't have specifications for PDF 3.0 yet. Do you have?
We'll discuss this in the PDFsharp team.

Modify the source code to return all versions found.

Or open the PDF file directly and read the header with your own code.
You can open a filestream once and use that for PDFsharp and your own purposes.

**Comment by DiegoFSFlor** [comment]
thanks for the feedback
